src/utils/validation/skill-schema.ts

```diff
@@ -0,0 +1,8 @@
+import { number, object, string } from "yup"
+
+export const createSkillSchema = object().shape({
+  name: string().required("Skill name is required"),
+  description: string().required("Description is required"),
```

Comment:
can be pls double check on ticket if this is required

Reply:
ows mb description is not required.